### PR TITLE
#feature - Greeting user as per time of the day.

### PR DESCRIPTION
**Pull request by yogesh-**
image

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -41,11 +44,16 @@ function App() {
       console.log(mycity, temp, "from API call", icon, imgAlt);
```
Comment (syed-shaida-hussain) [inline src/App.js:44]: You can remove console logs.

**Review by syed-shaida-hussain — APPROVED** [review]
just remove console logs . Rest looks good.

File: src/utilities/greet.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+export const Greet = () => {
+  const [greet, setGreet] = useState("");
+  useEffect(() => {
+    var time = new Date();
+    var windowTime = time.toLocaleString([], {
+      hour: "numeric",
+      hour12: false,
+    });
+    console.log(windowTime);
```
Comment (junaidshaikh-js) [inline src/utilities/greet.js:11]: remove console log

File: src/App.js (hunk 1 above)
Comment (yogesh-) [reply src/App.js:44]: ok

**Review by junaidshaikh-js — APPROVED** [review]
Good work. 

Remove console logs from the code. Can be merged.

**Pull request merged**